test/unit/gamma.js

```diff
@@ -48,9 +48,10 @@ describe('Gamma correction', function () {
     sharp(fixtures.inputPngOverlayLayer1)
       .resize(320)
       .gamma()
+      .jpeg()
       .toBuffer(function (err, data, info) {
         if (err) throw err;
-        assert.strictEqual('png', info.format);
+        assert.strictEqual('jpeg', info.format);
         assert.strictEqual(320, info.width);
         fixtures.assertSimilar(fixtures.expected('gamma-alpha.jpg'), data, { threshold: 20 }, done);
```

Comment:
I wonder if we can remove the `{ threshold: 20 }` bit now that you've updated this to (correctly) use JPEG output?

Reply:
Will try it out tomorrow.